src/cclib/parser/utils.py

```diff
@@ -77,30 +75,30 @@ def convertor(value, fromunits, tounits):
 
         "cm-1_to_eV":       lambda x: x / 8065.54429,
         "cm-1_to_hartree":  lambda x: x / 219474.6313708,
-        "cm-1_to_kcal":     lambda x: x / 349.7550112,
+        "cm-1_to_kcal_mol-1":     lambda x: x / 349.7550112,
```

Comment:
There were two votes in #461 for using `kcal_per_mole` (instead of `kcal_mol-1`). You should ask in that issue if anyone is opposed, and if not, use `kcal_per_mol` here and elsewhere.

Reply:
+1

Reply:
I did so because in codes for  ‘’per unit ‘’ there is ‘’unit-1’’

Reply:
Let's move this discussion to the main issue.

Reply:
Changed everything in af9fba77adc01d6c9a522aa4a3f54fc7db8aaf34, according to consensus from #461.